app/Http/routes.php

```diff
@@ -15,4 +15,13 @@
     return view('welcome');
 });
 
+Route::get('projects','ProjectsController@index');
+Route::get('projects/create','ProjectsController@create');
+Route::get('projects/{id}','ProjectsController@show');
+Route::post('projects/store','ProjectsController@store');
+Route::delete('projects/{id}','ProjectsController@destroy');
+Route::get('projects/{id}/edit','ProjectsController@edit');
+Route::post('projects/{id}/edited','ProjectsController@update');
+
```

Comment:
Routes should be initiated as Route::resource(balala) and it will do all this in 1 step
you can check the routes via php artisan route:list and adjust your url to the ones generated